crates/ruff/resources/test/fixtures/flake8_no_pep420/test_pass_script/script

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python3
```

Comment:
Note: it seems this fixture never gets tested, I guess because it doesn't have a `.py` or `.pyi` extension. Haven't researched what could/should be done about that yet.

Reply:
Nevermind, found it (and fixed related bugs!)